testing/mldb_python_plugin_test.cc

```diff
@@ -348,20 +348,20 @@ print "a"
     getResult = proxy.get("/v1/plugins/pyplugin_local_multifile/routes/func");
     cerr << getResult << endl;
 
-    BOOST_CHECK_EQUAL(getResult.code(), 200);
-    BOOST_CHECK_EQUAL(getResult.jsonBody()["value"].asString(), "in function");
+    BOOST_REQUIRE_EQUAL(getResult.code(), 200);
+    BOOST_REQUIRE_EQUAL(getResult.jsonBody()["value"].asString(), "in function");
 
     getResult = proxy.get("/v1/plugins/pyplugin_local_multifile/routes/doc");
     cerr << getResult << endl;
-    BOOST_CHECK_EQUAL(getResult.code(), 404);
+    BOOST_REQUIRE_EQUAL(getResult.code(), 500);
```

Comment:
404 must now be set by the plugin routes in their handler. Failing to set a return code is a 500 error.